orderer/common/multichannel/registrar.go

```diff
@@ -886,6 +915,15 @@ func (r *Registrar) joinSystemChannel(
 func (r *Registrar) RemoveChannel(channelID string) error {
 	r.lock.Lock()
 	defer r.lock.Unlock()
+
+	if removalStatus, ok := r.pendingRemoval[channelID]; ok {
+		if removalStatus {
+			return types.ErrChannelPendingRemoval
+		} else {
+			return types.ErrChannelRemovalFailure
```

Comment:
Why can't we "try again"?

Reply:
Maybe https://github.com/hyperledger/fabric/pull/2027#discussion_r515592791 this should only apply to the Joining a channel that experienced a removal failure.